doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -86,9 +87,18 @@ The picture below highlights the PMON vertical and its association with other lo
 ### DPU startup sequence diagram
 <p align="center"><img src="./images/dpu-startup-seq.svg"></p>
 
-### Configuring startup and shutdown
-* The DPUs can be powered down by configuring the admin_status as shown.
-* The corresponding switch configDB table is also shown
+#### 2.1.1 DPUs in dark mode
```

Comment:
@rameshraghupathy can you define what is DARK mode?
Also mention the default is DARM mode enabled

Reply:
> @rameshraghupathy can you define what is DARK mode? Also mention the default is DARM mode enabled

Done
@prgeor